statement.md

```diff
@@ -0,0 +1,84 @@
+
+### Compatibility with browsers and assistive technology
+
+JupyterLab is designed to be compatible with the following technologies:
+
+* Windows, macOS, iOS, Android; Firefox, Chrome, Safari, Chromium browsers (mobile and desktop).
+
+JupyterLab is not compatible with:
+
+* Internet Explorer, Edge; JAWS, NVDA, VoiceOver, Narrator, Orca screen readers; voice control technology.
+
+### Technical specifications
+
+Accessibility of JupyterLab relies on the following technologies to work with the particular combination of web browser and any assistive technologies or plugins installed on your computer:
+
+* HTML
+* WAI-ARIA
+* CSS
+* JavaScript
+
+These technologies are relied upon for conformance with the accessibility standards used.
+
+### Limitations and alternatives
+
+Despite our best efforts to ensure accessibility of JupyterLab, there may be some limitations. Below is a description of known limitations, and potential solutions. Please contact us if you observe an issue not listed below.
+
+Known limitations for JupyterLab:
+
+1. **Documents**: Documents writen by the community may not include accessible content because we do not and cannot review every document that can be opened in JupyterLab. To support accessible documents, we are drafting guidelines for accessible document content with an emphasis on Jupyter notebooks. Please report the issue to the author and [open an issue on jupyter/accessibility](https://github.com/jupyter/accessibility/issues/new) describing the problem and the behavior you expect so we may integrate it into our content guidelines.
+    
+2. **JupyterLab extensions**: JupyterLab extensions writen by the community may not be accessible because JupyterLab extensions can be written by anyone in the community and have no required review process. We do not and can not review every JupyterLab extension. To support accessible extensions, we encourage extension authors to use existing, accessible JupyterLab components for their extension. We also  provide periodic opportunities for community education on accessibility. Please report the issue to the author and let them know the [jupyter/accessibility](https://github.com/jupyter/accessibility/) community may be able to provide guidance.
+
+### Assessment approach
+
+Jupyter accessibility contributors assessed the accessibility of JupyterLab by the following approaches:
+
+* Self-evaluation
+* Regular automated testing to monitor for regressions (can be found at [jupyter/accessibility](https://github.com/jupyter/accessibility)
+* User feedback</li>
+
+### Evaluation report
+
+An evaluation for JupyterLab is available at: [jupyterlab/jupyterlab/issues/9399](https://github.com/jupyterlab/jupyterlab/issues/9399).
+
+### Evaluation report
+
+User reports on JupyterLab's accessibility are available at:[the jupyterlab/jupyterlab label `tag:accessibility`](https://github.com/jupyterlab/jupyterlab/labels/tag%3AAccessibility).
+
+### Formal complaints
```

Comment:
Merge with Feedback? Feedback and Formal Complaints?

Reply:
I think they are separate because formal complaints seems to be private and/or legal action related complaints, but we don't have a more formal channel at the moment so I'll put them together.